apps/crn-server/src/data-providers/users.ts

```diff
@@ -66,6 +48,7 @@ export default class Users implements UserDataProvider {
 
     if (isFullUpdate) {
       const existingUser = await this.userSquidexRestClient.fetchById(id);
+
```

Comment:
ad #44 - UserPAtchDataObject should be just UserUpdateDataObject

Reply:
Done